### PR TITLE
Add MyFRITZ!

### PR DESCRIPTION
**Pull request by phallobst**
Although the doc page refers to FRITZ!Box 7590, the MyFRITZ! login is device-independent.

### CODE REVIEW

File: _data/remote.yml [hunk 1]
```diff
@@ -85,6 +85,14 @@ websites:
       - totp
     doc: http://www.cc-onlinehelp.com/en/#index/administrati/users/authenticati/authenticati.html
 
+  - name: MyFRITZ!
+    url: https://sso.myfritz.net/
+    img: myfritz.png
+    tfa:
+      - totp
+      - hardware
+    doc: https://en.avm.de/service/fritzbox/fritzbox-7590/knowledge-base/publication/show/3427
```
Comment (Carlgo11) [inline _data/remote.yml:95]: Is there any other doc page (maybe in German?) that better explains how to enable 2FA?

**Comment by phallobst** [comment]
I haven't found any better documentation, but here's a screenshot of the setup process:

tfa-myfritz

The "Proof of Personal Identity" also supports using a U2F key, so I added that as well. I suspect that fingerprint and face recognition works on iOS devices only, so I could not check those.

File: _data/remote.yml (hunk 1 above)
Comment (phallobst) [reply _data/remote.yml:95]: (see below)

**Review by DeanR97 — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

File: _data/remote.yml [hunk 2]
```diff
@@ -85,6 +85,15 @@ websites:
...
     doc: http://www.cc-onlinehelp.com/en/#index/administrati/users/authenticati/authenticati.html
 
+  - name: MyFRITZ!
+    url: https://sso.myfritz.net/
+    img: myfritz.png
+    tfa:
+      - totp
+      - hardware
+      - u2f
+    doc: https://en.avm.de/service/fritzbox/fritzbox-7590/knowledge-base/publication/show/3427
```
Comment (ApCoder123) [inline _data/remote.yml:95]: I think that this doc page might be slightly more useful - it mentions both U2F and TOTP at the bottom of the page and has better details on how to enable
```suggestion
    doc: https://en.avm.de/service/fritzbox/fritzbox-7590/knowledge-base/publication/show/3602
```

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
Superseded by #5703

**Pull request closed without merging**